python/xacc.py

```diff
@@ -406,9 +406,19 @@ def benchmark(xacc_settings):
 
     results_name = "%s_%s_out" % (os.path.splitext(
         tail)[0], timestr)
-    f = open(results_name+".ab", 'w')
-    f.write(str(buffer))
-    f.close()
+    
+    if 'output_logs' not in xacc_settings['Benchmark']:
```

Comment:
I prefer this to work the other way - by default log files should be written, and you can specify not doing it in the input file or input dictionary.